internal/config/projectconfig/apply.go

```diff
@@ -0,0 +1,43 @@
+package projectconfig
+
+import (
+	"fmt"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/commitdev/zero/internal/util"
+	"github.com/commitdev/zero/pkg/util/flog"
+)
+
+// Apply will bootstrap the runtime environment for the project
+func Apply(dir string, projectContext *ZeroProjectConfig, applyEnvironments []string) []string {
+	flog.Infof(":tada: Bootstrapping project %s. Please use the zero.[hcl, yaml] file to modify the project as needed. %s.", projectContext.Name)
+
+	flog.Infof("Cloud provider: %s", "AWS") // will this come from the config?
+
+	flog.Infof("Runtime platform: %s", "Kubernetes")
+
+	flog.Infof("Infrastructure executor: %s", "Terraform")
+
+	// other details...
+
+	return makeAll(dir, projectContext, applyEnvironments)
+}
+
+func makeAll(dir string, projectContext *ZeroProjectConfig, applyEnvironments []string) []string {
+	environmentArg := fmt.Sprintf("ENVIRONMENT=%s", strings.Join(applyEnvironments, ","))
+	envars := []string{environmentArg}
```

Comment:
Could you also add the parameters from the project config as env vars?

Reply:
done